internal/databases/postgres/bguploader.go

```diff
@@ -189,7 +189,8 @@ func (b *BgUploader) processFiles(fileChan <-chan string) {
 		b.started[name] = struct{}{}
 		if err := b.workerCountSem.Acquire(b.ctx, 1); err == nil {
 			go func() {
-				uploadedFile := b.upload(name)
+				// todo Should it use b.ctx? It will interrupt last file upload and change behavior.
```

Comment:
I guess we should not use b.ctx here. The default behavior is to wait for all uploads to finish.